src/main.js

```diff
@@ -470,30 +481,42 @@ function TimekitBooking() {
   // Render the powered by Timekit message
   var renderPoweredByMessage = function(pageTarget) {
 
+    var campaignName = 'widget'
+    var campaignSource = window.location.hostname.replace(/\./g, '-')
+    if (config.widgetSlug) campaignName = 'hosted-widget'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'campaignName'.](https://www.codacy.com/app/laander/booking-js/file/2386876808/issues/source?bid=2969108&fileBranchId=3228106#l487)